src/components/BaseLink.vue

```diff
@@ -14,13 +15,17 @@ defineProps<{
     v-if="typeof link === 'string'"
     :href="link"
     target="_blank"
-    class="whitespace-nowrap"
+    :class="`whitespace-nowrap ${disabled ? 'pointer-events-none' : ''}`"
```

Comment:
```suggestion
    :class="['whitespace-nowrap', { 'pointer-events-none': disabled }]"
```

Reply:
Maybe you missed this in vue docs 😉

Reply:
I was aware of it, but it wasn't working when I tried it first, but that could be some other issue, I will fix it.